### PR TITLE
Release notes for SAM 2021.12.02 Revision 2

### PR DESCRIPTION
**Pull request by cpaulgilman**
All PRs from SAM and SSC for 2021.12.02 Patch 2:

https://github.com/NREL/SAM/pulls?q=is%3Apr+is%3Aclosed+milestone%3A%222021.12.02+Patch+2%22

https://github.com/NREL/ssc/pulls?q=is%3Apr+is%3Aclosed+milestone%3A%222021.12.02+Patch+2%22

### CODE REVIEW

File: doc/releasenotes.html [hunk 1]
```diff
@@ -59,6 +59,88 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
 
 -->
 
+<h1 id="2021-12-02-r2">Version 2021.12.02 Revision 2: June 6 , 2022</h1>
+
+<p>This is a maintenance update to <a href="#2021-12-02">SAM 2021.12.02 Revision 1</a>.</p>
+
+<ul>
+<li>Improve feeback for parametric simulation graphs when they cannot be displayed. (<a href="https://github.com/NREL/SAM/pull/1051">1051</a>)</li>
```
Comment (brtietz) [inline doc/releasenotes.html:67]: Spelling error in "feedback"

**Review by brtietz — COMMENTED** [review]
Thanks for putting this together! I spotted two typos - marked on individual lines.

File: doc/releasenotes.html (hunk 1 above)
Comment (cpaulgilman) [reply doc/releasenotes.html]: Thanks! Fixed in https://github.com/NREL/SAM/pull/1070/commits/fb1cb15b7c84f98458336dc53c0e3f0fa1d614f3.

File: doc/releasenotes.html [hunk 2]
```diff
@@ -59,6 +59,88 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
...
+<li>Fix Solmetric Suneye shading import. (<a href="https://github.com/NREL/SAM/pull/1052">1052</a>)</li>
+<li>Add transmission losses to loss diagram. (<a href="https://github.com/NREL/SAM/pull/1000">1000</a>, <a href="https://github.com/NREL/ssc/pull/790">SSC 790</a>)</li>
+<li>Make 6parsolve module coefficient calculator error messages more descriptive. (<a href="https://github.com/NREL/ssc/pull/827">SSC 827</a>, <a href="https://github.com/NREL/ssc/pull/802">802</a>)</li>
+<li>Improve handling of albedo data errors. (<a href="https://github.com/NREL/ssc/pull/799">SSC 799</a>)</li>
+<li>Report time series outputs over lifetime to facilitate comparison with other outputs in results. (<a href="https://github.com/NREL/ssc/pull/791">SSC 791</a>)</li>
+</ul>
+</li>
+<li>Battery Storage
+<ul>
+<li>For REopt battery sizing and dispatch API calls. (<a href="https://github.com/NREL/SAM/pull/999">999</a>)</li>
```
[the remark this answers is not in the document]
Comment (cpaulgilman) [reply doc/releasenotes.html]: https://github.com/NREL/SAM/pull/1070/commits/fb1cb15b7c84f98458336dc53c0e3f0fa1d614f3

**Pull request merged**